apps/mull-ui/src/app/pages/messages/direct-message/direct-message-list/direct-message-list.tsx

```diff
@@ -0,0 +1,77 @@
+import { faEllipsisH } from '@fortawesome/free-solid-svg-icons';
+import { History } from 'history';
+import React, { useContext, useState } from 'react';
+import {
+  Friend,
+  Post,
+  useCreateDirectMessageChannelMutation,
+  useFriendsQuery,
+  User,
+} from '../../../../../generated/graphql';
+import { ContactRow, CustomTextInput } from '../../../../components';
+import UserContext from '../../../../context/user.context';
+import './direct-message-list.scss';
+
+export interface DirectMessageListPageProps {
+  history: History;
+}
+
+const DirectMessageListPage = ({ history }: DirectMessageListPageProps) => {
+  const [searchValue, setSearchValue] = useState('');
+  const { userId } = useContext(UserContext);
+  const { data, loading } = useFriendsQuery();
+  const [createDirectMessageChannel] = useCreateDirectMessageChannelMutation();
+
+  const buildLastMessage = (latestPost: Post, friend: Partial<Friend>): string => {
+    let lastMessage: string;
+    if (!latestPost) lastMessage = `Start chatting with ${friend.name}!`;
+    else if (latestPost.media)
+      lastMessage = `${latestPost.user.id === userId ? 'You' : friend.name} sent an image.`;
+    else if (latestPost.message && !latestPost.media)
+      lastMessage = `${latestPost.user.id === userId ? 'You:' : ''} ${latestPost.message}`;
```

Comment:
Past a certain amount of characters, we should just have `...` at the end. Like `Becky is trying to sell MLM produ...`

![image](https://user-images.githubusercontent.com/33991121/113947225-2d14cc80-97d8-11eb-9689-991efdf78bcf.png)

Reply:
Please use the ExpandableText component for this 👀